static/src/stylesheets/module/content/live-blog/navigation.scss

```diff
@@ -42,82 +39,103 @@
 
 .liveblog-navigation__link {
     display: inline-block;
+    position: relative;
+    color: colour(neutral-1);
+    background: transparent;
+    border: 1px solid colour(neutral-4);
+    height: $gs-baseline * 3;
+    line-height: 38px;
+    font-weight: bold;
+    vertical-align: middle;
```

Comment:
Removed `background: transparent`. Heights hardcoded to fit alongside other large buttons such as toast, The buttons will grow with a larger text size though :smiley: